src/context/RealmsContext.tsx

```diff
@@ -33,7 +35,18 @@ export const RealmsProvider: FunctionComponent = ({ children }) => {
   }
 
   useFetch(
-    () => adminClient.realms.find({ briefRepresentation: true }),
+    async () => {
+      try {
+        return await adminClient.realms.find({ briefRepresentation: true });
+      } catch (error) {
+        const e = error as Error | AxiosError;
+        if ("response" in e && e.response && e.response.status < 500) {
+          return [];
+        } else {
+          throw error;
+        }
```

Comment:
You can use `axios.isAxiosError()` here to determine the error is indeed an `AxiosError`. You can also omit the `else` clause since the `if` branch terminates with a `return` statement. The `Error` type can also be omitted, as it's fine to re-throw a value of the type `unknown`.

```suggestion
        if (
          axios.isAxiosError(error) &&
          error.response &&
          error.response.status < 500
        ) {
          return [];
        }

        throw error;
```